tests/sentry/integrations/github/test_integration.py

```diff
@@ -383,3 +383,49 @@ def test_get_message_from_error(self):
             == base_error
             + f" Please also confirm that the commits associated with the following URL have been pushed to GitHub: {url}"
         )
+
+    @responses.activate
+    def test_github_prevent_install_until_pending_deletion_is_complete(self):
+        self._stub_github()
+        # First installation should be successful
+        self.assert_setup_flow()
+        integration = Integration.objects.get(provider=self.provider.key)
+        oi = OrganizationIntegration.objects.get(
+            integration=integration, organization=self.organization
+        )
+        # set installation to pending deletion
+        oi.status = ObjectStatus.PENDING_DELETION
+        oi.save()
+
+        # New Installation
+        self.installation_id = "install_2"
+
+        self._stub_github()
+
+        resp = self.client.get(
+            "{}?{}".format(self.init_path, urlencode({"installation_id": self.installation_id}))
+        )
+
+        assert (
+            b'{"success":false,"data":{"error":"GitHub installation pending deletion."}}'
+            in resp.content
+        )
+        assert (
+            b"It seems that your Sentry organization has an installation pending deletion. Please wait ~15min for the uninstall to complete and try again."
```

Comment:
@NisanthanNanthakumar Is there a way to directly reference `integration-pending-deletion.html` in tests? Maybe check if that was the rendered response file? If not, you can totally leave this, it's just frustrating when changing copy breaks tests sometimes.

Reply:
👍  yea we can use `assertTemplateUsed`